server/src/main/java/umm3601/digitalDisplayGarden/PlantController.java

```diff
@@ -187,10 +188,9 @@ else if(rating.get("like").equals(false))
             }
         }
 
+        interactions = likes + dislikes + comments;
 
-        out.put("commentCount", comments);
-        out.put("likeCount", likes);
-        out.put("dislikeCount", dislikes);
+        out.put("interactionCount", interactions);
```

Comment:
We should update the comment on this method to reflect the change in the output.